portfolio.html

```diff
@@ -217,7 +358,10 @@ <h6 class="connect">LET'S CONNECT</h6>
             </li>
           </ul>
         </div>
-        <input type="button" class="more" value="GET MY RESUME" >
+        <div class="seeMore">
+          <button class="more">GET MY RESUME</button>
+        </div>
+      </header>
         <ul class="dropdownMenu" >
```

Comment:
- [ :thinking: ] In my opinion it would be better to stick to the design as much as possible as recommended by the project requiremens, so, I suggest to give `li` a fixed(responsive as this goes for both mobile and desktop version) width then justify the text and the arrow space-between to separate them.
 - Your design:
![langYours](https://user-images.githubusercontent.com/97834160/206594772-31a06897-699b-4ebe-b1c3-171655e65032.png)


 - Figma design:
![langFig](https://user-images.githubusercontent.com/97834160/206594800-d977aa02-f039-41ed-a86f-8ab0fafcfa1f.png)